funding/manager.go

```diff
@@ -3983,6 +4055,22 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 		maxHtlcs = f.cfg.RequiredRemoteMaxHTLCs(capacity)
 	}
 
+	// Prepare the optional channel fee values from the initFundingMsg.
+	// If useBaseFee or useFeeRate are false the client did not
+	// provide fee values hence we assume default fee settings from
+	// the config.
+	forwardingPolicy := htlcswitch.ForwardingPolicy{
+		BaseFee: f.cfg.DefaultRoutingPolicy.BaseFee,
+		FeeRate: f.cfg.DefaultRoutingPolicy.FeeRate,
+	}
+	if baseFee != nil {
+		forwardingPolicy.BaseFee = lnwire.MilliSatoshi(*baseFee)
+	}
+
+	if feeRate != nil {
+		forwardingPolicy.FeeRate = lnwire.MilliSatoshi(*feeRate)
```

Comment:
if we change this to PPM we need to be sure that we change it to MilliSatoshis :+1:

Reply:
Sorry I am not sure I understand this comment.

`forwardingPolicy.FeeRate` is a `uint64` currently denoting the fee rate in milli satoshis. Since the `forwardingPolicy` doesn't have a fee field expressed in ppm this one is used to work around it. Are you suggesting that we add a new field for ppm to `ForwardingPolicy`?

Reply:
Sorry that was confusing. I just meant that we should avoid phrasing this in the api (proto) as "The effective fee rate in milli-satoshis."

 This is a uint64 that means parts per million (independently of how we treat it inside lnd). We use `lnwire.MilliSatoshi` to represent it because we want to multiply it by the amount (that is `lnwire.MiliSatoshi`), but if you say `feeRate=1` you would get 1 btc if you forward one milion btc, the same way you would get 1 sat if you foward 1M sats and so on.